projects/1.18/assets/untamedwilds/untamedwilds/lang/zh_cn.json

```
@@ -0,0 +1,726 @@
+  "entity.untamedwilds.tortoise_marginated": "缘翘陆龟",
+  "entity.untamedwilds.tortoise_marginated.sciname": "Testudo marginata",
+  "entity.untamedwilds.tortoise_star": "印度星斑陆龟",
+  "entity.untamedwilds.tortoise_star.sciname": "Geochelone elegans",
+  "entity.untamedwilds.tortoise_sulcata": "苏卡达龟",
+  "entity.untamedwilds.tortoise_sulcata.sciname": "Centrochelys sulcata",
+  
+  "item.untamedwilds.snake_spawn_egg": "蛇刷怪蛋",
+  "item.untamedwilds.material_snake_skin": "蛇蜕",
+  "item.untamedwilds.egg_snake": "蛇蛋",
+  "entity.untamedwilds.snake": "蛇",
+  "entity.untamedwilds.snake_adder": "极北蝰",
+  "entity.untamedwilds.snake_adder.sciname": "Vipera berus",
+  "entity.untamedwilds.snake_ball_python": "球蟒",
+  "entity.untamedwilds.snake_ball_python.sciname": "Python regius",
+  "entity.untamedwilds.snake_black_mamba": "黑曼巴蛇",
+  "entity.untamedwilds.snake_black_mamba.sciname": "Dendroaspis polylepis",
+  "entity.untamedwilds.snake_carpet_python": "地毯蟒",
+  "entity.untamedwilds.snake_carpet_python.sciname": "Morelia spilota",
+  "entity.untamedwilds.snake_cave_racer": "黑眉锦蛇",
+  "entity.untamedwilds.snake_cave_racer.sciname": "Elaphe taeniura",
+  "entity.untamedwilds.snake_coral": "德州珊瑚蛇",
+  "entity.untamedwilds.snake_coral.sciname": "Micrurus fulvius",
+  "entity.untamedwilds.snake_coral_blue": "蓝长腺丽纹蛇",
+  "entity.untamedwilds.snake_coral_blue.sciname": "Calliophis bivirgatus",
+  "entity.untamedwilds.snake_corn": "玉米锦蛇",
+  "entity.untamedwilds.snake_corn.sciname": "Pantherophis guttatus",
+  "entity.untamedwilds.snake_emerald": "翡翠树蚺",
+  "entity.untamedwilds.snake_emerald.sciname": "Corallus caninus",
+  "entity.untamedwilds.snake_grass_snake": "草蛇",
+  "entity.untamedwilds.snake_grass_snake.sciname": "Natrix natrix",
+  "entity.untamedwilds.snake_gray_kingsnake": "灰带王蛇",
+  "entity.untamedwilds.snake_gray_kingsnake.sciname": "Lampropeltis alterna",
+  "entity.untamedwilds.snake_green_mamba": "东非绿曼巴蛇",
+  "entity.untamedwilds.snake_green_mamba.sciname": "Dendroaspis angusticeps",
+  "entity.untamedwilds.snake_rattlesnake": "森林响尾蛇",
+  "entity.untamedwilds.snake_rattlesnake.sciname": "Crotalus horridus",
+  "entity.untamedwilds.snake_rice_paddy": "铅色水蛇 ",
+  "entity.untamedwilds.snake_rice_paddy.sciname": "Hypsiscopus plumbea",
+  "entity.untamedwilds.snake_swamp_moccasin": "食鱼蝮",
+  "entity.untamedwilds.snake_swamp_moccasin.sciname": "Agkistrodon piscivorus",
+  "entity.untamedwilds.snake_taipan": "内陆太攀蛇",
+  "entity.untamedwilds.snake_taipan.sciname": "Oxyuranus microlepidotus",
+  "entity.untamedwilds.snake_western_rattlesnake": "大草原响尾蛇",
+  "entity.untamedwilds.snake_western_rattlesnake.sciname": "Crotalus viridis",
+  "untamedwilds.subtitles.snake.warning": "蛇：嘶嘶",
+  "untamedwilds.subtitles.snake.rattle": "蛇：响尾声",
+
+  "item.untamedwilds.large_snake_spawn_egg": "巨蟒刷怪蛋",
+  "item.untamedwilds.egg_large_snake": "巨蟒蛋",
+  "entity.untamedwilds.large_snake": "巨蟒",
+  "entity.untamedwilds.large_snake_anaconda": "绿水蚺",

```

```suggestion
  "entity.untamedwilds.large_snake_anaconda": "森蚺",
```
[百度](https://baike.baidu.com/item/%E7%BB%BF%E6%A3%AE%E8%9A%BA/793681?fr=ge_ala)，iNaturalist和维基百科将其直接称作森蚺

---

iNaturalist[https://www.inaturalist.org/taxa/32213-Eunectes-murinus]有叫绿水蚺的（？）
百度给出的是绿森蚺、森蚺和绿水蟒的，还有叫亚马逊森蚺的

---

i

---

iNaturalist [https://www.inaturalist.org/taxa/32213-Eunectes-murinus] 有叫绿水蚺的（？） 百度给出的是绿森蚺、森蚺和绿水蟒的，还有叫亚马逊森蚺的

---

iNaturalist https://www.inaturalist.org/taxa/32213-Eunectes-murinus 有叫绿水蚺的（？） 
百度给出的是绿森蚺、森蚺和绿水蟒的，还有叫亚马逊森蚺的